deploy/frontend.yml

```diff
@@ -18,24 +19,52 @@ objects:
         paths:
           - /apps/learning-resources
       image: ${IMAGE}:${IMAGE_TAG}
-      navItems:
-        - appId: learningResources
-          title: Learning Resources
-          href: "/openshift/learning-resources"
-        - appId: learningResources
-          title: Learning Resources
-          href: "/ansible/learning-resources"
-        - appId: learningResources
-          title: Learning Resources
-          href: "/insights/learning-resources"
-        - appId: learningResources
-          title: Learning Resources
-          href: "/edge/learning-resources"
-        - appId: learningResources
-          title: Learning Resources
-          href: "/iam/learning-resources"
+      feoConfigEnabled: true
+      bundleSegments:
+        - segmentId: learning-resources-openshift
+          bundleId: openshift
+          position: 2000
```

Comment:
What is position for btw?

Reply:
Ok cool, that's what I thought. If all 4 of the bundles have the same position is the order random then or just in the same order? Maybe we'd want it to increment 2000, 2001, 2002, 2003..

Reply:
They aren't displayed in the same place, so them having the same position shouldn't be a problem.